### PR TITLE
Implement Planner Messages

### PR DESCRIPTION
**Pull request by epicstar**
## Description

From PlanetaryRobotics/moonranger-ground-backend#205, send and receive Planner app messages.

Do not review until:
* this is no longer a draft
* #98 is merged

## How has this been tested?

TODO

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have read and followed the `CONTRIBUTING.md` file
- [ ] I have applied the .clang-format file to my changes
- [ ] I have commented my code using Doxygen style comments (see Contributing MD for examples), particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the design documentation 
- [ ] I have updated any relevant parameters in the internal ICD
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] If this is a bug-fix, I have added tests that can catch similar bugs in the future
- [x] New and existing unit tests pass locally with my changes
- [ ] For cfe apps I have reserved an empty address for my msgs_ids and perf_ids from the [tracking list](https://docs.google.com/spreadsheets/d/1yRUhNTRBOiGLswlsWftNaYlwGKDeNQ_fCnDYFrc_7fU/edit#gid=772812381) and checked that it does not clash with other apps
- [ ] For new added message types, there is a timestamp included via either an explicit timestamp variable in the struct or buried in the a tlmHeader.

### CODE REVIEW

**Comment by tenzincrouch** [comment]
the prior PR has been merged, if that was what was holding this back you can update it now.

**Comment by epicstar** [comment]
@zhaowenzi and @tenzincrouch. There is a merge conflict here. It would be nice if I could get assistance to fix the merge conflicts. I don't have any idea which branch @zhaowenzi 's commits are coming from.

File: message-utilities/include/message_builder.h [hunk 1]
```diff
@@ -60,6 +61,14 @@ typedef union {
 
     // command messages
     MOONRANGER_Teleop_Cmd_t Teleop_Cmd;
+
+    PLANNER_Eval_Trigger_t Planner_Eval_Trigger_Cmd;
+    PLANNER_HkTlm_t PlannerHk_Tlm;
```
Comment (ashwin21rao) [inline message-utilities/include/message_builder.h:66]: Please move line 66 to the section above (for telemetry messages).
Other than this, LGTM.

**Review by ashwin21rao — CHANGES_REQUESTED** [review]

**Comment by epicstar** [comment]
@ashwin21rao thank you for the review. Please check again (I resolved the convo.

**Review by ashwin21rao — APPROVED** [review]
LGTM

**Pull request merged**